processor/transformprocessor/README.md

```diff
@@ -43,6 +45,8 @@ processors:
         - set(status.code, 1) where attributes["http.path"] == "/health"
         - keep_keys(resource.attributes, "service.name", "service.namespace", "cloud.region")
         - set(name, attributes["http.route"])
+        - truncate(attributes, 4096)
+        - truncate(resource.attributes, 4096)
```

Comment:
"truncateAll"? Since I expect to have a truncate for a specific key?

Reply:
Good thinking.  I will update the name.